products/views.py

```diff
@@ -1,3 +1,23 @@
 from django.shortcuts import render
 
 # Create your views here.
+from django.http  import JsonResponse
+from django.views import View
+
+from products.models import *
+
+class RecommendedView(View):
+    def get(self, request, product_id):
+        try:
+            category_id         = Product.objects.get(id = product_id).category
+            recommend_list = [{
+                'name'      : value.name,
+                'image'     : [image.url for image in value.productimage_set.all()],
+                'skintype'  : [types.skin_type.name for types in value.productskintype_set.all()]
+                } for value in Product.objects.filter(category = category_id).exclude(id=product_id)]
+
+            return JsonResponse({'result' : [ recommend_list ] }, status = 200)
```

Comment:
애초에 리스트로 만들었는데 굳이 리스트로 또 감싸준 이유는 무엇인가요?
```suggestion
            return JsonResponse({'result' : recommend_list }, status = 200)
```

Reply:
업보청산으로 인한 휴먼 에러입니다...